src/components/file/AddFile.tsx

```diff
@@ -6,22 +6,33 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { useState } from "react";
 import AddFilePopup from "@components/user/AddFile";
 import { File as PrismaFile } from "@prisma/client";
+import React from "react";
 
 const AddFile = ({
   seniorId,
   seniorFolder,
   files,
+  editFile,
+  setEditFile,
 }: {
   seniorId: string;
   seniorFolder: string;
   files: PrismaFile[];
+  editFile?: PrismaFile;
+  setEditFile: React.Dispatch<React.SetStateAction<PrismaFile | undefined>>;
```

Comment:
I don't think this is too clean, but it makes the most sense given the time constraint.